components/dashboard/src/user-settings/SelectIDE.tsx

```diff
@@ -76,8 +81,14 @@ export default function SelectIDE(props: SelectIDEProps) {
             <div className="w-112 max-w-full my-4">
                 <SelectIDEComponent
                     onSelectionChange={actuallySetDefaultIde}
+                    pinnedEditorVersions={
+                        isOrgOwnedUser && orgSettings?.pinnedEditorVersions
+                            ? new Map(Object.entries(orgSettings.pinnedEditorVersions))
+                            : undefined
+                    }
```

Comment:
Does this was discussed recently? Last time for pin editor I remember we discussed and it was fine not to show the pinned version here

Reply:
> Does this was discussed recently?

Nope, I thought this part is missing in preview PR 😅, if it's already discussed to not show, I will revert this change

Reply:
Addressed https://github.com/gitpod-io/gitpod/pull/19542/commits/0b350c22aab7890f21bf1349ffdcd6f7816321ce